spec/system/courses/review_spec.rb

```diff
@@ -401,6 +401,53 @@
       expect(page).to have_text(target_l2.title)
     end
 
+    context 'when the course has inactive students' do
+      let(:inactive_team) do
+        create :startup, level: level_1, access_ends_at: 1.day.ago
+      end
+      let!(:submission_from_inactive_team) do
+        create(
+          :timeline_event,
+          :with_owners,
+          latest: true,
+          owners: inactive_team.founders,
+          target: team_target
+        )
+      end
+
+      scenario 'coach can access inactive submission', js: true do
+        sign_in_user course_coach.user, referrer: review_course_path(course)
+
+        expect(page).to have_content('Showing all 7 submissions')
+
+        # Search by email
+        fill_in 'filter', with: 'Inactive Students'
+        click_button 'Pick Include: Inactive Students'
+
+        expect(page).to have_content('Showing all 8 submissions')
+        expect(page).to have_content(inactive_team.name)
+      end
+    end
+
+    context 'when random filters are applied' do
+      let(:random_level) { create :level, :one }
+      let(:random_target) { create :target, :for_founders }
+
+      scenario 'coach visits review dashboard', js: true do
+        sign_in_user course_coach.user,
+                     referrer:
+                       review_course_path(
+                         course,
+                         levelId: random_level.id,
+                         targetId: random_target.id
+                       )
+
+        expect(page).to have_content('Showing all 7 submissions')
+        expect(page).not_to have_content(random_level.name)
+        expect(page).not_to have_content(random_target.title)
+      end
+    end
+
```

Comment:
This appears to be a dupe of what's in #778.

Reply:
Will be resolved once the other is merged with master. I branched out from the other as it was touching mostly the same files.